### PR TITLE
Added release notes for Serverless TP 1.3.0

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-rn-1-3-0.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+[id="serverless-rn-1-3-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.3.0
+
+[id="new-features-1-3-0_{context}"]
+== New features
+* {ServerlessProductName} has been updated to use Knative Serving 0.10.0.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.10.0.
+
```
Comment (markusthoemmes) [inline modules/serverless-rn-1-3-0.adoc]: Any reason why this doesn't include HTTPs and/or the bug fixes we've listed in the Changelog?

**Comment by mvinkler** [comment]
QE approved.

File: modules/serverless-rn-1-3-0.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-3-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.3.0
```
Comment (markusthoemmes) [inline modules/serverless-rn-1-3-0.adoc]: One last general bit: We're dropping support for OCP 4.1. That'll ship with https://github.com/openshift-knative/serverless-operator/pull/57 and is the last thing blocking the release currently.

**Comment by mgencur** [comment]
QE Aproved

**Review by bergerhoffer — APPROVED** [review]
LGTM for peer review!

**Comment by bergerhoffer** [comment]
As mentioned in the other PR, best to start w/ master when the changes apply to the latest (4.3). Cherrypicking to master and 4.3

**Pull request merged**